microceph/ceph/rgw.go

```diff
@@ -153,3 +190,17 @@ func symlinkRGWKeyring(keyPath, ConfPath string) error {
 
 	return nil
 }
+
+func writeFile(path, data string, mode int) error {
```

Comment:
Could the std lib function os.WriteFile() be used here instead?

Reply:
Yes. I updated the code to use that on https://github.com/canonical/microceph/pull/355/commits/80f44405cea0d2d9206976a3bc5a524e742a0f11.